about.php

```diff
@@ -0,0 +1,48 @@
+<?php
+require 'dbConn.php';
+function retrieve_data($db)
+{
+    $retrieve_statement = $db->prepare('SELECT `content`, `picture_link`, `phone_number`, `email` FROM `about_me` WHERE `id` = 1;');
+    $retrieve_statement->execute();
+    $start_data = $retrieve_statement->fetchAll();
+    return $start_data;
+}
+$start_data = retrieve_data($db);
+?>
+<!DOCTYPE html>
+<html>
+<head>
+
+    <title> About Page </title>
+
+</head>
+<body>
+
+<form action="db_querys.php" method="post">
+
+    About me content
+    <input type="text" name="content"  value="<?php echo $start_data[0]['content']?>"> <br>
```

Comment:
If you fetch instead of fetch all you won't have to use [0] - same comment for all lines with [0]